### PR TITLE
feat: carbon champions announcement

### LINKED ISSUE #13887: Make a new GitHub Action for posting slack block kit messages
We currently have some block kit messages being sent to slack via Github Actions workflows defined in the repo. We'd like to improve upon this to be able to send custom slack block kit messages to slack via the Github Action repo.

The ideal flow would be that someone could 
1. Design a slack block kit message in the [slack block kit builder](https://app.slack.com/block-kit-builder)
1. Manually invoke/run a GitHub action
    1. Provide their own `text` value
    1. Provide their own `channel` value (this can be a single string or an array of strings)
    1. Provide their own `blocks` definition by copying and pasting from the slack block kit builder
1. Running the action would post their message to the channels provided

---

- [x] Create a new workflow.yml file
- [x] Copy the contents from `.github/workflows/slack-office-hours-dev.yml` to the new file
- [x] Remove the schedule/cron trigger
- [x] Add a workflow_dispatch trigger so it can be run manually
- [ ] Add inputs for the `blocks`, `text`, and `channel` parameters so these can be configured when the action is ran from the GitHub UI
- [ ] I think this can be tested by providing a `@username` slack handle in the `channel` field to send yourself a test message
- [ ] Verify that copying and pasting a slack block kit message works - sometimes the values from the slack block kit builder are not valid JSON, we may need to provide some way to sanitize or convert the slack block kit to valid JSON

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #13887

Create action specifically for internal Carbon Champ reminders sent to #carbon-team

- This action should automatically run on the 2nd Wednesday of every month

### CODE REVIEW

File: .github/workflows/carbon-champs-reminder.yml
Comment (tay1orjones) [inline .github/workflows/carbon-champs-reminder.yml:1]: What do you think about renaming this file to `slack-carbon-champs-reminder.yml` instead? I think it's nice to have all the slack workflows next to one another when organized alphabetically like in the vscode sidebar.

**Review by tay1orjones — APPROVED** [review]
Looks good to me, just one minor suggestion 👍

File: .github/workflows/carbon-champs-reminder.yml
Comment (andreancardona) [reply .github/workflows/carbon-champs-reminder.yml:1]: perfect :)

File: .github/workflows/slack-carbon-champs-reminder.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Slack Office Hours Reminder - Dev
```
Comment (alisonjoseph) [inline .github/workflows/slack-carbon-champs-reminder.yml]:
```suggestion
name: Carbon Champions Nomination Reminder
```

**Review by alisonjoseph — APPROVED** [review]
Small suggestion to update the name, but LGTM!

**Pull request merged**